### ISSUE TITLE
yolov10n.onnx conversion issue

### ISSUE THREAD

**Issue by happyiminjay1** [kaylorchen/rk3588-yolo-demo#23]
Thanks for the great work.

I found this quite strange in the yolov10n.onnx shared on the google drive.

![image](https://github.com/user-attachments/assets/a4629355-5f69-4ee1-bbfd-2097aff7dd26)

Is it reversed?

Thanks

**Comment by kaylorchen** [comment]
> Thanks for the great work.
> 
> I found this quite strange in the yolov10n.onnx shared on the google drive.
> 
> ![image](https://private-user-images.githubusercontent.com/42690774/374377214-a4629355-5f69-4ee1-bbfd-2097aff7dd26.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgzNTM2MTMsIm5iZiI6MTcyODM1MzMxMywicGF0aCI6Ii80MjY5MDc3NC8zNzQzNzcyMTQtYTQ2MjkzNTUtNWY2OS00ZWUxLWJiZmQtMjA5N2FmZjdkZDI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDA4VDAyMDgzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQyMWRhMDdjMDkxMThhMTFhODQ3YWJhODkxNDMxMTI1ZGNmZjdmM2ZhN2I5MWMwNGE2MmRmYWE0YjZjNWQwNWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.J23ZUPMIaBW2bppcNHxCH2Yt3uo6Y1NRxYx3DS8OKLU)
> 
> Is it reversed?
> 
> Thanks

you are right. I have updated my yolov10's export.py. you can export your model by https://github.com/kaylorchen/yolov10/tree/rk3588 .
